### PR TITLE
Repeat week

### PR DESCRIPTION
**Pull request by Arturgouveia1970**
in this pull request, work was made on the margins of the document.
general adjustments to the styling of the page.

### CODE REVIEW

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Arturgouveia1970, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
- [ ] [OPTIONAL] Kindly give the branch a meaningful name, not `repeat-week` like the name of the project `mobile-version`.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -7,40 +7,130 @@
...
+        <h2 id="skills"><img class="rect3" id="skill" src="images/Rectangle57.png" alt="Rectangle">Skills</h2>
+        <ul class="skills">
+          <li class="skills1">CodeKit</li>
+          <li class="skills2">Github</li>
+          <li class="skills3">Codepen</li>
+          <li class="skills4">Gitlab</li>
+          <li class="skills5">Terminal</li>
+        </ul>
+      </div>
+    </section>
```
[the remark this answers is not in the document]
Comment (Arturgouveia1970) [reply index.html:131]: @gemmen29 , please I need help in how to remove this white space. I have tried @media and it does not change. there isn't one element that exceeds 375px of width. any suggestions?

[the remark this answers is not in the document]
Comment (gemmen29) [reply index.html:131]: Yes, most of the problems come from this part
image
try to change `margins` with `paddings` and get rid of fixed `widths` in that section

File: index.html [hunk 2]
```diff
@@ -7,40 +7,130 @@
...
+          </ul>
+        <h2 id="frames"><img class="rect2" src="images/Rectangle56.png" alt="rectangle">Frameworks</h2>
+        <ul class="frames">
+          <li class="frames1">Bootstrap</li>
+          <li class="frames2">Ruby on Rails</li>
+          <li class="frames3">RSPec</li>
+          <li class="frames4">CapyBara</li>
+          <li class="frames5">Selenium</li>
+        </ul>
+        <h2 id="skills"><img class="rect3" id="skill" src="images/Rectangle57.png" alt="Rectangle">Skills</h2>
```
[the remark this answers is not in the document]
Comment (Arturgouveia1970) [reply index.html]: linters tells me that I can't have ul as a child of ul. don't understand...

**Review by sja-thedude — COMMENTED** [review]
## Invalid Code Review Request ❌

Hi @Arturgouveia1970, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't pass all linters.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by CollinsTatang — APPROVED** [review]
Hi @Arturgouveia1970 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**